src/widgets/auto-style/histogram.js

```diff
@@ -12,7 +12,9 @@ var HistogramAutoStyler = AutoStyler.extend({
   },
 
   getColorLine: function (sym) {
-    var shape = this.dataviewModel.getDistributionType();
+    var shape = this.dataviewModel.getDistributionType(
+      this.dataviewModel.getUnfilteredDataModel().get('data')
```

Comment:
one comment, I don't know if unfiltered data model will have the whole data histogram when the dashboard stats with a filter (or other widget has a filter). I think it worths checking